lib/cache/sqlite-cache-store.js

```diff
@@ -112,9 +114,9 @@ class SqliteCacheStore {
         staleAt INTEGER NOT NULL
       );
 
-      CREATE INDEX IF NOT EXISTS idx_cacheInterceptorV1_url ON cacheInterceptorV1(url);
-      CREATE INDEX IF NOT EXISTS idx_cacheInterceptorV1_method ON cacheInterceptorV1(method);
-      CREATE INDEX IF NOT EXISTS idx_cacheInterceptorV1_deleteAt ON cacheInterceptorV1(deleteAt);
+      CREATE INDEX IF NOT EXISTS idx_cacheInterceptorV${VERSION}_url ON cacheInterceptorV${VERSION}(url);
+      CREATE INDEX IF NOT EXISTS idx_cacheInterceptorV${VERSION}_method ON cacheInterceptorV${VERSION}(method);
+      CREATE INDEX IF NOT EXISTS idx_cacheInterceptorV${VERSION}_deleteAt ON cacheInterceptorV${VERSION}(deleteAt);
```

Comment:
What does this do? What's the `idx_` prefix?

Reply:
@flakey5 ?

Reply:
> What does this do?

As in `CREATE INDEX ...`? It creates map of a column to an id in the table. Helps speed up reads when we do a `SELECT ... WHERE ...` but at the cost of some write performance (since it writes to the table and the indices)

> What's the `idx_` prefix?

Short for index, it's just a naming convention